docs/octopus-rest-api/octopus.server.exe-command-line/index.md

```diff
@@ -22,7 +22,7 @@ hideInThisSection: true
 - **[import-certificate](/docs/octopus-rest-api/octopus.server.exe-command-line/import-certificate.md)**:  Replace the certificate that Octopus Server uses to authenticate itself with its Tentacles.
 - **[license](/docs/octopus-rest-api/octopus.server.exe-command-line/license.md)**:  Import a license key.
 - **[list-instances](/docs/octopus-rest-api/octopus.server.exe-command-line/list-instances.md)**:  Lists all installed Octopus instances.
-- **[lost-master-key](/docs/octopus-rest-api/octopus.server.exe-command-line/lost-master-key.md)**:  Get your Octopus Server working again after losing your Master Key.
+- **[lost-master-key](/docs/octopus-rest-api/octopus.server.exe-command-line/lost-master-key.md)**:  Get your Octopus Server working again after losing your master key.
```

Comment:
@wordlee - looks like this got fixed and then reverted?
https://github.com/OctopusDeploy/OctopusDeploy/commit/a1c9a09b7d2995e3e8572ca45edf732a207a7e81

Reply:
Yeah, that was late on a Friday and I was worried about failing test over the weekend. I'm redoing it now and will tag you on the PR.